lib/wibox/layout/fixed.lua

```diff
@@ -110,6 +110,14 @@ function fixed:get_children()
     return self.widgets
 end
 
+--- Replace the layout children
+-- @tparam table children A table composed of valid widgets
+function fixed:set_children(children)
+    if not children then return self:reset() end
```

Comment:
This check isn't needed here nor in any of the other implementations of `:set_children()`. The API documentation says this must be a table, and the implementation of `setup()` below only calls this with a table.